### PR TITLE
moved react tutorial, removed playground, updated examples page

### PR DESCRIPTION
**Pull request by micahgodbolt**

### CODE REVIEW

**Comment by micahgodbolt** [comment]
this was a quick bit of commits. happy to take feedback and continue to tweak

File: docs/content/docs/start/tutorial.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: 'Tutorial: Create a Fluid Framework application'
+title: 'Tutorial: Create a Fluid app'
```
Comment (skylerjokiel) [inline docs/content/docs/start/tutorial.md:2]:
```suggestion
title: 'Tutorial: Hello world walkthrough'
```

Comment (micahgodbolt) [reply docs/content/docs/start/tutorial.md:2]: going with capital W since it's more of a title than a sentence

**Review by skylerjokiel — COMMENTED** [review]
Please delete and remove the other react file.

**Review by skylerjokiel — APPROVED** [review]
🦦

File: docs/content/docs/start/tutorial.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: 'Tutorial: Create a Fluid Framework application'
+title: 'Tutorial: Hello World walkthrough'
```
Comment (tylerbutler) [inline docs/content/docs/start/tutorial.md:2]:
```suggestion
title: 'Tutorial: A simple Fluid application'
```

I don't love my suggestion.

File: docs/themes/thxvscode/layouts/partials/footer.html [hunk 3]
```diff
@@ -24,19 +24,20 @@
                 <ul class="links">
                     <li><a id="footer-privacy-link" href="https://privacy.microsoft.com/privacystatement"
                             target="_blank">Privacy</a></li>
-                    <li><a id="footer-terms-link"
-                            href="https://www.microsoft.com/legal/terms-of-use"
+                    <li><a id="footer-terms-link" href="https://www.microsoft.com/legal/terms-of-use"
```
Comment (tylerbutler) [inline docs/themes/thxvscode/layouts/partials/footer.html:27]: nit: why?

**Review by tylerbutler — APPROVED** [review]
Revert the react tutorial move since that's in my PR #7416 already but looks great otherwise!

File: docs/themes/thxvscode/layouts/partials/footer.html (hunk 3 above)
Comment (micahgodbolt) [reply docs/themes/thxvscode/layouts/partials/footer.html:27]: vs code prettier probably :D

**Pull request merged**